File: Assistencia-Tecnica-app/src/app/_Models/Empresa.ts
import { Usuario } from './Usuario';

// export interface Empresa {
export class Empresa {
    id: number;
    razaoSocial: string;
    nomeFantasia: string;
    cnpj: string;
    setor: string;
    endereco: string;
    email: string;
    eelefone: string;
    ativo: boolean;
    dataAlteracao: Date | string;
    alteradoPor: string;

    usuarios: Usuario[];
}

File: AssistenciaTecnica.Domain/Empresa.cs
using System;
using System.Collections.Generic;

namespace AssistenciaTecnica.Domain
{
    public class Empresa
    {
        public int Id { get; set; }
        public string RazaoSocial { get; set; }
        public string NomeFantasia { get; set; }
        public string Cnpj { get; set; }
        public string Setor { get; set; }
        public string Endereco { get; set; }
        public string Email { get; set; }
        public string Telefone { get; set; }
        public bool Ativo { get; set; }
        public DateTime DataAlteracao { get; set; }
        public string AlteradoPor { get; set; }
    
        public ICollection<Usuario> Usuarios { get; set; }
    }



}